pipeline/manual_e2e_test.py

```diff
@@ -53,21 +53,21 @@
 #
 # These files contain real sample data, usually 4 measurements each, the first 2
 # are measurements that succeeded, the last two are measurements that failed.
-def local_data_to_load_1(*_) -> List[str]:
+def local_data_to_load_1(*_) -> List[str]:  # type: ignore
```

Comment:
nit: maybe we can name these local_data_to_load_incrememental and local_data_to_load or something like that?

Reply:
switched the names to match the types of data being loaded